bin/yebash

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-LD_PRELOAD=$DIR/../lib/libyebash.so /bin/bash
+LD_PRELOAD=$DIR/../lib/libyebash.so /bin/bash --rcfile <(echo 'unset LD_PRELOAD; . ~/.bashrc')
```

Comment:
Accordng to the POSIX - yes (http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html chapter 2.6.1)